recipes/msaboot/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: msaboot
+  version: "0.1.0"
+
+source:
+  url: https://github.com/phac-nml/msaboot/archive/v0.1.0.tar.gz
+  sha256: 41a34899b4c768abec4bf4e15d5a0a08451208fab9403b00454f1b7203971fa6
+
+build:
+  number: 0
+  skip: True # [win]
+
+requirements:
+  build:
+    - python
+    - numpy
+    - biopython
```

Comment:
You can remove `numpy` and `biopython` from the build requirements, and please add `setuptools` instead.

Reply:
Thank you. Does setuptools have to be listed first or is the order that I have listed okay/